src/database/migrations/z1629758770-update-stories-trigger.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // if (process.env.NODE_ENV === 'test') {
+    //   return;
+    // }
+
+    return queryInterface.sequelize.query(`
+      CREATE OR REPLACE FUNCTION update_loves_stories()
+    RETURNS TRIGGER AS $$
+    BEGIN
+      IF (TG_OP = 'INSERT') THEN -- INSERT operations
+          UPDATE story_engagement SET loves = loves + 1 WHERE "contentId"=NEW."contentId";
+          RETURN NEW;
+      ELSEIF (TG_OP = 'DELETE') THEN -- DELETE operations
+          UPDATE story_engagement SET loves = loves - 1 WHERE "contentId"=OLD."contentId";
+          RETURN OLD;
+      END IF;
+    END;
+    $$ LANGUAGE plpgsql;
+
+    DROP TRIGGER IF EXISTS update_loves_stories
```

Comment:
No need to drop it. See above, it says `CREATE OR REPLACE FUNCTION`.
You can remove this drop and the create trigger. Unless you are changing when the trigger is triggered (`BEFORE INSERT OR DELETE`) or the table, which I think is not the case.

Reply:
you're right, I made the change